src/gfnff/gfnff_eg.f90

```diff
@@ -126,6 +127,8 @@ subroutine gfnff_eg(env,pr,n,ichrg,at,xyz,makeq,g,etot,res_gff, &
       ebatm=0
       eext =0
 
+      eatoms = 0.0d0
```

Comment:
Maybe describe the use and how the energy is calculated a little bit further here.